bin/node/src/service.rs

```diff
@@ -140,6 +144,72 @@ pub fn new_partial(
     })
 }
 
+#[allow(clippy::type_complexity)]
+#[allow(clippy::too_many_arguments)]
```

Comment:
maybe let's stay consistent: either use clippy silencer instead of `ConsensusPartyParams` or create a new struct instead of these silencers

Reply:
I am actually consistent here (in this file we already had such silencer) :P

Reply:
In finality-aleph we should not use such silencers though imho

Reply:
bleh